cluster/saltbase/salt/kube-apiserver/kube-apiserver.manifest

```diff
@@ -54,6 +54,11 @@
 {% set token_auth_file = "--token_auth_file=/dev/null" -%}
 {% set basic_auth_file = "" -%}
 
+{% set min_request_timeout = "" -%}
+{% if grains.test is defined -%}
+  {% set min_request_timeout = "--min-request-timeout=300" %}
```

Comment:
So instead of having this logic in salt, you'd have a CONN_TIMEOUT variable or something in the config-test.sh/config-default.sh files.